[2021-02-28 13:02:51,264] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:01:38.396028+00:00 [queued]>
[2021-02-28 13:02:51,269] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:01:38.396028+00:00 [queued]>
[2021-02-28 13:02:51,269] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 13:02:51,269] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2021-02-28 13:02:51,269] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 13:02:51,274] {taskinstance.py:1063} INFO - Executing <Task(HttpSensor): is_api_available> on 2021-02-28T12:01:38.396028+00:00
[2021-02-28 13:02:51,276] {standard_task_runner.py:52} INFO - Started process 39952 to run task
[2021-02-28 13:02:51,282] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'bitcoin_price_dag', 'is_api_available', '2021-02-28T12:01:38.396028+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bitcoin_price_dag.py', '--cfg-path', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmpb7q_g2n4', '--error-file', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmp32iyf9wf']
[2021-02-28 13:02:51,283] {standard_task_runner.py:77} INFO - Job 8: Subtask is_api_available
[2021-02-28 13:02:51,311] {logging_mixin.py:104} INFO - Running <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:01:38.396028+00:00 [running]> on host 120.0.168.192.in-addr.arpa
[2021-02-28 13:02:51,332] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=bitcoin_price_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2021-02-28T12:01:38.396028+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-28T12:01:38.396028+00:00
[2021-02-28 13:02:51,333] {http.py:102} INFO - Poking: ticker
[2021-02-28 13:02:51,336] {base.py:65} INFO - Using connection to: id: bitcoin_api. Host: http://blockchain.info, Port: None, Schema: , Login: , Password: None, extra: None
[2021-02-28 13:02:51,338] {http.py:140} INFO - Sending 'GET' to url: http://blockchain.info/ticker
[2021-02-28 13:02:51,503] {local_task_job.py:146} INFO - Task exited with return code Negsignal.SIGABRT
